File: AspNetMvcUnleashed/0672329980_source/Chapter11/VB/MvcApplication1/MvcApplication1/Controllers/RemoveController.vb
Public Class RemoveController
    Inherits Controller

    <OutputCache(Duration:=9999, VaryByParam:="None", Location:=OutputCacheLocation.Server)> _
    Public Function Time() As ActionResult
        Return View()
    End Function

    Public Function Clear() As ActionResult
        HttpResponse.RemoveOutputCacheItem("/Remove/Time")
        Return RedirectToAction("Time")
    End Function

End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter11/CS/MvcApplication1/MvcApplication1/Controllers/RemoveController.cs
using System.Web;
using System.Web.Mvc;
using System.Web.UI;

namespace MvcApplication1.Controllers
{
    public class RemoveController : Controller
    {
        [OutputCache(Duration=9999, VaryByParam="None", Location=OutputCacheLocation.Server)]
        public ActionResult Time()
        {
            return View();
        }

        public ActionResult Clear()
        {
            HttpResponse.RemoveOutputCacheItem("/Remove/Time");
            return RedirectToAction("Time");
        }


    }
}
